projects/packages/forms/src/contact-form/css/grunion.css

```diff
@@ -317,23 +312,27 @@
  * handled by flex-basis and max-width.
  */
 .wp-block-jetpack-contact-form  :not(.is-layout-constrained, .is-layout-flow, .is-layout-flex, .is-layout-grid) {
-	.grunion-field-width-25-wrap {
+	> .grunion-field-width-25-wrap {
```

Comment:
Without the direct descendent selector, the `:not` doesn't seem to work. I guess because it can match any element in the hierarchy of elements, and when that happens, even if the parent has `is-layout-flex` it'll still apply this style.

Not sure if there's a better way to do this.